packages/cactus-plugin-ledger-connector-besu/src/test/typescript/benchmark/run-plugin-ledger-connector-besu-benchmark.ts

```diff
@@ -0,0 +1,239 @@
+  // const firstHighNetWorthAccount = besuTestLedger.getGenesisAccountPubKey();
+
+  // keychainPlugin for signed transactions
+  const { privateKey } = Secp256k1Keys.generateKeyPairsBuffer();
+  const keyHex = privateKey.toString("hex");
+  const pem = keyEncoder.encodePrivate(keyHex, KeyFormat.Raw, KeyFormat.PEM);
+  const signedKeychainPlugin = new PluginKeychainMemory({
+    instanceId: uuidv4(),
+    keychainId: keychainIdForSigned,
+    backend: new Map([[keychainRefForSigned, pem]]),
+    logLevel,
+  });
+
+  // keychainPlugin for unsigned transactions
+  const keychainEntryValue = testEthAccount1.privateKey;
+  const unsignedKeychainPlugin = new PluginKeychainMemory({
+    instanceId: uuidv4(),
+    keychainId: keychainIdForUnsigned,
+    backend: new Map([[keychainRefForUnsigned, keychainEntryValue]]),
+    logLevel,
+  });
+  unsignedKeychainPlugin.set(
+    HelloWorldContractJson.contractName,
+    JSON.stringify(HelloWorldContractJson),
+  );
+
+  const pluginRegistry = new PluginRegistry({
+    plugins: [signedKeychainPlugin, unsignedKeychainPlugin],
+  });
+
+  const options: IPluginLedgerConnectorBesuOptions = {
+    instanceId: uuidv4(),
+    rpcApiHttpHost,
+    rpcApiWsHost,
+    pluginRegistry,
+    logLevel,
+  };
+  const connector = new PluginLedgerConnectorBesu(options);
+  pluginRegistry.add(connector);
+
+  const expressApp = express();
+  expressApp.use(bodyParser.json({ limit: "250mb" }));
+  const server = http.createServer(expressApp);
+
+  const wsApi = new SocketIoServer(server, {
+    path: Constants.SocketIoConnectionPathV1,
+  });
+
+  const listenOptions: IListenOptions = {
+    hostname: "127.0.0.1",
+    port: 0,
+    server,
+  };
+  const addressInfo = (await Servers.listen(listenOptions)) as AddressInfo;
+  test.onFinish(async () => await Servers.shutdown(server));
+  const { address, port } = addressInfo;
+  const apiHost = `http://${address}:${port}`;
+  // t.comment(
+  //   `Metrics URL: ${apiHost}/api/v1/plugins/@hyperledger/cactus-plugin-ledger-connector-besu/get-prometheus-exporter-metrics`,
+  // );
+
+  // const wsBasePath = apiHost + Constants.SocketIoConnectionPathV1;
+  // t.comment("WS base path: " + wsBasePath);
+  const besuApiClientOptions = new BesuApiClientOptions({ basePath: apiHost });
+  const apiClient = new BesuApiClient(besuApiClientOptions);
+
+  await installOpenapiValidationMiddleware({
+    logLevel,
+    app: expressApp,
+    apiSpec: OAS,
+  });
+
+  await connector.getOrCreateWebServices();
+  await connector.registerWebServices(expressApp, wsApi);
+
+  return {
+    httpApi: apiClient,
+    apiServer: connector,
+  };
+};
+
+const main = async (opts: { readonly argv: Readonly<Array<string>> }) => {
+  const logLevel: LogLevelDesc = "WARN";
+
+  const gitRootPath = path.join(
+    __dirname,
+    "../../../../../../", // walk back up to the project root
+  );
+
+  console.log("%s gitRootPath=%s", LOG_TAG, gitRootPath);
```

Comment:
Use logger instead of `console.log`

Reply:
Hello @outSH , I have looked into the Cacti repository and found multiple logger (./util/util, log4js, hyperledger/cactus-common). I would like to ask which logger specifically do you mean in this requested change? Thank you.

Reply:
@ruzell22 Hello, please use the one from `@hyperledger/cactus-common` :)